base_station/gui/src/components/PDBData.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+   <div class = "wrapper">
+       <div v-bind:style="{color: activeColor}">
+        <span>PDB Current Draw: {{currentDraw}} amps</span><br>
+        <span>PDB Temperature: {{temperature}} C°</span>
+       </div>
+   </div> 
+</template>
+
+<script>
+import SpectralData from './SpectralData.vue'
+export default {
+  components: { SpectralData },
+    data(){
+        return{
+            currentDraw:0,
+            temperature:0,
+            activeColor:"red"
+        }
+    },
+    created: function(){
+        setTimeout(() => {  this.currentDraw = 36.6;

```

Comment:
idk what this but is this from the mock values

Reply:
We don't use this file anymore, should be using PDBFuse.vue

Reply:
probs should take that out..